daffodil-cli/src/test/scala-cli/edu/illinois/ncsa/daffodil/debugger/TestCLIDebugger.scala

```diff
@@ -285,6 +285,18 @@ class TestCLIdebugger {
 
     try {
       val cmd = String.format("%s -d parse -s %s -r matrix %s", Util.binPath, testSchemaFile, testInputFile)
+
+      //
+      // If you want to run a test in the Eclipse debugger instead of by
+      // spawning a shell, here's the idiom.
+      //
+      // Note that this doesn't do the expect stuff. You have to type the
+      // commands at the console pane, and inspect the output to see if it is
+      // what you want.
+      //
+      // val args = cmd.split(' ').tail // cmdline minus the daffodil program name.
+      // Main.run(args) // Run Main directly, so we can debug.
+      //
```

Comment:
This comments probably belongs somewhere else, maybe at the top of TestCLIDebugger or in a CLI readme or something?